pkg/controller/job-controller.go

```diff
@@ -200,7 +200,7 @@ func (c *JobController) SyncHandler(key string) (Result, error) {
 			Namespace: namespace,
 		},
 	}
-	err := c.k8sClient.Get(ctx, client.ObjectKeyFromObject(&jobCR), &jobCR)
+	err = c.k8sClient.Get(ctx, client.ObjectKeyFromObject(&jobCR), &jobCR)
```

Comment:
Go would be so much safer if they would just disallow to redefine a variable within the same scope...

Reply:
Just a rant against the Go language :-)